### PR TITLE
Updating External Resources: Add tutorial in Portuguese and using PyGMT in Google Colab

### PR DESCRIPTION
**Pull request by andrebelem**
**This is an update for External Resources**

- a PyGMT webinar conducted in Portuguese

As suggested in the [forum](https://forum.generic-mapping-tools.org/t/pygmt-very-short-course-in-portuguese/5028), I am applying to include this webinar in Portuguese for PyGMT.

**Preview**: https://pygmt-dev--3360.org.readthedocs.build/en/3360/external_resources.html

### CODE REVIEW

**Review by yvonnefroehlich — COMMENTED** [review]
@andrebelem Cool, this is a nice addition to this collection and especially interesting for the non-English speaking world. Thanks 🙂!

File: doc/external_resources.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ to submit a pull request with your recommended addition to the
 
 :::::{grid} 1 2 2 3
 
+::::{grid-item-card} 2024 PyGMT Webinar using Google Colab (in Portuguese)
+:link: [[https://www.generic-mapping-tools.org/egu22pygmt](https://github.com/andrebelem/Oficina_PyGMT)/](https://github.com/andrebelem/Oficina_PyGMT)
```
[the remark this answers is not in the document]
Comment (andrebelem) [reply doc/external_resources.md:16]: Oh yeah ! I made a copy-paste which went wrong haha

Comment (andrebelem) [inline doc/external_resources.md:16]: Humm...how can I change this ? The construction is wrong. It should be:
link: [https://github.com/andrebelem/Oficina_PyGMT](https://github.com/andrebelem/Oficina_PyGMT)

[the remark this answers is not in the document]
Comment (yvonnefroehlich) [reply doc/external_resources.md:16]: Yeah I thought this, as there is the link form the EGU 2022 short course included...

Are you able / allowed to click the "Commit suggestion" button, which is just below my comment? Then you can apply my suggestion directly.

Comment (yvonnefroehlich) [reply doc/external_resources.md:16]: This looks like my suggestion from the code review. Are you able / allowed to click the "Commit suggestion" button, which is just below my inital review comment? Then you can apply my suggestion directly:

github_review_commit_suggestion

Of course you can also make a new commit with the correct link and push to this branch.

Comment (andrebelem) [reply doc/external_resources.md]: Yeap. I did it.

Comment (yvonnefroehlich) [reply doc/external_resources.md]: Great! Should be good now https://pygmt-dev--3360.org.readthedocs.build/en/3360/external_resources.html.

Comment (yvonnefroehlich) [reply doc/external_resources.md]: Please update the title to make it more descriptive.

Comment (yvonnefroehlich) [reply doc/external_resources.md]: I am sorry, my comment was probably confusing 🙁. It's only about the title of the PR. You can change it via the "edit" bottom on the right:
title_pr

Maybe something like "External Resources: Add tutorial in Portuguese "

Comment (andrebelem) [reply doc/external_resources.md]: Perfect. I changed the title. Thanks.

**Review by yvonnefroehlich — APPROVED** [review]
Thanks for providing these resources!

**Review by seisman — APPROVED** [review]

**Pull request merged**